src/libraries/Microsoft.Extensions.Options/src/OptionsFactory.cs

```diff
@@ -90,6 +94,9 @@ public TOptions Create(string name)
         /// <summary>
         /// Creates a new instance of options type
```

Comment:
Not part of your PR but needs a tiny fix:
```suggestion
        /// Creates a new instance of options type.
```